.dependabot/config.yml

```diff
@@ -0,0 +1,9 @@
+version: 1
+update_configs:
+  - package_manager: "python"
+    directory: "/"
+    update_schedule: "monthly"
```

Comment:
For Antenna, this is the big one we want help with. This will keep Python dependencies up to date. The one we don't want to update (for now) is Falcon. I'll add a thing later to prevent dependabot from updating that.